Library/Formula/logstash.rb

```diff
@@ -2,19 +2,20 @@
 
 class Logstash < Formula
   homepage 'http://logstash.net/'
```

Comment:
I'd suggest updating this to this permanent redirect from above URL:
https://www.elastic.co/products/logstash